src/components/GameCard/GameCard.jsx

```diff
@@ -1,17 +1,22 @@
 import React from 'react'
 import styles from './GameCard.module.css'
 import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+
+const GameCard = ({ id, title, img }) => {
+    const defaultImageUrl = 'https://via.placeholder.com/250x250' // TODO: move to another file with constants
```

Comment:
const Greet = ({firstName = 'John', lastName = 'Doe'}) => {
  return (<div>{`Hi ${firstName} ${lastName}`}!</div>)
}

For you TODO!

Reply:
got it